app/controllers/companies_controller.rb

```diff
@@ -14,7 +16,6 @@ def index
   end
 
   def show
-    @company = Company.find(params[:id])
```

Comment:
Why leave this out?

Reply:
Isnt it logical to also allow for a specific company to be found? It's also left out in the skip_before_action so it should be in here as well right?

Reply:
@pimmesz I like it dry 😏